client/src/models/testSuiteModels.ts

```diff
@@ -36,7 +36,10 @@ export interface Result {
 
 export interface TestInput {
   name: string;
+  title?: string;
   value?: string;
+  type?: string;
+  description?: string;

```

Comment:
We can save it for a later PR if you want, but I think we'll want to be able to set default values sooner rather than later.

Reply:
Not sure if this is what you meant, but I made it so type defaults to 'text' unless specified.

Reply:
We should definitely save it for a later PR because I want to get this in. I meant test writers should be able to specify default values for inputs, like how we have default scopes in program.